extension/content/utils/api.js

```diff
@@ -97,6 +97,13 @@ export default class NormandyAPI {
       version,
     });
   }
+
+  async fetchActions({ version = 3 }) {
```

Comment:
there is no other version of action endpoint so we don't need the option here and you don't need to pass version into `request` down below.